dts/arm/nxp/nxp_lpc54xxx.dtsi

```diff
@@ -30,6 +31,13 @@
 	};
 
 	soc {
+		clkctl0: clkctl@40000000 {
+			compatible = "nxp,lpc-syscon";
+			reg = <0x40000000 0x4000>;
+			label = "CLKCTL_0";
```

Comment:
`SYSCON_0`?

Reply:
I have used CLKCTL as the newer SYSCON (found in RT600) have multiple CLKCTL blocks for clock control

Reply:
But this is the LPC54xxx, not the RT600?

Reply:
Done, I have changed this to syscon